### ISSUE TITLE
SASS files fail with error that file is circular dependency of itself

### ISSUE THREAD

**Issue by kirill-konshin** [aackerman/circular-dependency-plugin#55]
```
ERROR in Circular dependency detected:
src/skin/_styles.scss -> src/skin/_styles.scss
```

Sass file itself is 100% okay.

The error occurred with Webpack 5.

I am using `style-loader` -> `css-loader` -> `fast-sass-loader`.

**Comment by aackerman** [comment]
Thanks for the report, I'll do my best to take a look into this, but since it's not JS I won't make any promises. 

I would probably add `exclude: /scss/` to my configuration to get around this issue so it isn't a blocker.

**Comment by sindrisig** [comment]
I was trying out webpack 5 and saw this same error for .css & .less files. The only js files that popped up only after updating were files that used modules.exports so that might be something worth investigating

**Comment by aackerman** [comment]
I believe this is fixed in 5.2.2. I'm going to go ahead and close. If you're still seeing this issue I'll reopen.
